core/cost-model.md

```diff
@@ -0,0 +1,176 @@
+# Cost Model
+
+## Cost Model Constants
+
+- **TXN_SIGNATURE_COST**: 720  
+  Cost for each signature included in the transaction
+
+- **ED25519_SIGNATURE_COST**: 2400  
```

Comment:
there are two version of ed25519 cost, `2280` for original, `2400` for *strict* version, which is behind its own feature gate